### PR TITLE
Documenter update

### PR DESCRIPTION
**Pull request by abhro**
- Update Documenter.jl versions
- Rerun benchmarks
- Refresh Manifest.toml
- Move large docstring to doc page

### CODE REVIEW

**Comment by abhro** [comment]
Failing tests are from lack of compat for Statistics.jl, resolved in #17, and failure to deploy (permission issues)

**Comment by giordano** [comment]
Deplyment key updated: http://juliaastro.org/PSFModels.jl/dev/

File: docs/src/introduction.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+```
+
+### Plotting
+
+Finally, we provide plotting recipes (`psfplot`/`psfplot!`) from
+[RecipesBase.jl](https://github.com/JuliaPlots/RecipesBase.jl), which can be
+seen in use in the [API/Reference](@ref) section.
+
+```julia
+using PSFModels # hide
```
Comment (giordano) [inline docs/src/introduction.md:149]: This isn't hidden

Comment (abhro) [reply docs/src/introduction.md:149]: Oh. That was from reverting the example block. Will take care of that now.

Comment (abhro) [reply docs/src/introduction.md:149]: Fixed in #18! Thanks!

**Pull request merged**